content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Getting started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to get up and running with Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+To get started with Optimize, you will need the following:
+
+- A [Prisma Data Platform workspace](https://console.prisma.io).
+- A project that uses [Prisma Client](/orm/prisma-client) `5.16.1` or higher (we always recommend using the latest version of Prisma).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, or MongoDB database.
```

Comment:
Are you sure?

Reply:
> - A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, or MongoDB database.
Yes
![image](https://github.com/user-attachments/assets/6c0211ad-2acc-45d2-b040-44de59482d3b)

@mhessdev

Reply:
@ankur-arch You may want to add MS SQL Server as well to the list.